app/models/season/phase_switcher.rb

```diff
@@ -4,6 +4,16 @@ def self.season
     Season.current
   end
 
+  def self.switcher(phase)
+    @phases = {
+        'Proposals' => :fake_proposals_phase,
+        'Application' => :fake_application_phase,
+        'CodingSummer' => :fake_coding_phase,
+        'Real Time' => :back_to_reality
```

Comment:
If you want to stick with the "magic string" mapping, `Real Time` must be changed to `RealTime`